skripchenkoveta/hometask-5/4-8.sql

```diff
@@ -0,0 +1 @@
+DELETE FROM film CASCADE WHERE rating = 'PG-13'; 
```

Comment:
this statement causes error 
`update or delete on table "film" violates foreign key constraint "film_actor_film_id_fkey" on table "film_actor"`